bc-wikitemplate.md

```diff
@@ -91,90 +102,48 @@ _Each start should take less than 7 seconds_
-
-## Sync
-
-- [ ] Verify you are able to sync two devices using the secret code
-- [ ] Visit a site on device 1 and change shield setting, ensure that the saved site preference is synced to device 2
-- [ ] Enable Browsing history sync on device 1, ensure the history is shown on device 2
-- [ ] Clear browsing history on device 1, ensure the history is sync back on device 1 from device 2
-- [ ] Import/Add bookmarks on device 1, ensure it is synced on device 2
-- [ ] Ensure imported bookmark folder structure is maintained on device 2
-- [ ] Ensure bookmark favicons are shown after sync
-
-## Cookie and Cache
-
-- [ ] Go to http://samy.pl/evercookie/ and set an evercookie. Check that going to prefs, clearing site data and cache, and going back to the Evercookie site does not remember the old evercookie value. (No custom code in Brave for this yet)
-
-## Update tests
-
-- [ ] TODO: Test that updating using `BRAVE_UPDATE_VERSION=0.8.3` env variable works correctly.
-
-## Things to check once:
+- [ ] Test that https://mixed-script.badssl.com/ shows up as grey not red (no mixed content scripts are run)
 
-- Native ledger library, is it still stored in json
-- Migrate snaps to QA from devops
-- Can QA get production server copy for ledger back-end?
-- Can QA get more control to restart ledger services, when things go wrong.
-- Check one time that these exceptions work: `brave/common/shield_exceptions.cc`
-- Check one referrer blocking works properly with TLD+1
-- Evercookie works correctly?
-- Find a good channel (maybe youtube) to see upcoming Chrome issues that we should be concerned about.
-- Release notes should mention big things we are getting for free.
+
+### Cookie and Cache
+
+- [ ] Go to http://samy.pl/evercookie/ and set an evercookie. Check that going to prefs, clearing site data and cache, and going back to the Evercookie site does not remember the old evercookie value.
+
+### Session storage
+
+- [ ] Temporarily move away your `~/Library/Application\ Support/BraveSoftware/Brave-Browser/` and test that a new profile is created when browser is launched (`%userprofile%\appdata\Local\BraveSoftware\Brave-Browser\` in Windows, `./config/BraveSoftware/Brave-Browser/` in Ubuntu)
+- [ ] Test that windows and tabs restore when closed, including active tab.
+  - [ ] Ensure that the tabs in the above session are being lazy loaded when the session is restored
+
+## Chromium upgrade tests
+
+#### Adblock
+
+- [ ] Verify referrer blocking works properly for TLD+1 
+
+#### Components
 - Check once `brave/common/brave_switches.cc`
-- What happens if you install uBlock or ABP?
-- What happens with automatically installed extensions? 
+__________
+
+## Yet to implement
+
+- Keyboard Shortcuts
+- Ledger/Rewards
+  - Ledger Media (To be verified on YouTube and Twitch)
+- Sync
+- Tor private tabs
+- Manual Update tests from older version
+  - Test that updating using `BRAVE_UPDATE_VERSION=0.8.3` env variable works correctly.
```

Comment:
Is this valid in this codebase? I don't think so.

Reply:
No just kept it for reference to verify for offline installer once that is available